### ISSUE TITLE
Pop from empty list

### ISSUE THREAD

**Issue by diginfo** [joBr99/nspanel-lovelace-ui#204]
Getting this error in the logs and the panel is un-responsive to touch.

maybe line 149 needs to check if list is empty before popping :-)

```
2022-05-22 16:35:06.590839 WARNING nspanel-1: Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/appdaemon/threading.py", line 950, in worker
    funcref(args["event"], data, self.AD.events.sanitize_event_kwargs(app, args["kwargs"]))
  File "/config/appdaemon/apps/nspanel-lovelace-ui/luibackend/mqttListener.py", line 58, in mqtt_event_callback
    self._controller.button_press(entity_id, btype, value)
  File "/config/appdaemon/apps/nspanel-lovelace-ui/luibackend/controller.py", line 149, in button_press
    self._current_card = self._previous_cards.pop()
IndexError: pop from empty list
```

**Comment by joBr99** [comment]
hey, 
try to restart appdaemon this should fix it

however yeah, I should still add an check for this :)

**Comment by diginfo** [comment]
Thanks, yep, just thought I would report the issue.

Also, I added this to the forum, not sure if you saw it ??

```
[@jobraun](https://community.home-assistant.io/u/jobraun) - Thanks so much, the best solution I have tried so far :slight_smile:

And the fact that the display is updated whenever you save the apps.yaml file is really neat.

Is there a way to set a background colour or wallpaper image ?

I can’t seem to get the cardQR to work, what exactly is iText ?

it would be great if you could release your ‘sandbox’ configs that you created the screen sample images from, that would make getting started easier with the ability to copy out snippets from the sandbox without having to start from scratch.
```

**Comment by joBr99** [comment]
didn't check the forum today

**Comment by joBr99** [comment]
See #177 for background color/image

TLDR is that it isn't possible due to the lack of transparency support of nspanel and also it isn't possible to transfer pictures during runtime

However there is a partly implemented feature to change the colors on the screensaver #99

**Comment by diginfo** [comment]
Thanks for that.

How about the QRCARD, the ITEXT and releasing the app.yaml for your screenshot "Test Page" 1-6

Thanks again

**Comment by joBr99** [comment]
answered you in the forum

**Comment by diginfo** [comment]
just saw it, thanks

**Comment by joBr99** [comment]
there should not really be a difference between my test config an the one in the docs / in the repo

https://github.com/joBr99/nspanel-lovelace-ui/blob/main/appdaemon/apps.yaml

**Comment by diginfo** [comment]
Thank, I am talking about this: https://raw.githubusercontent.com/joBr99/nspanel-lovelace-ui/main/doc-pics/screens.png
